src/chrome/content/rules/StatusPage.io.xml

```diff
@@ -4,34 +4,12 @@
 		- status-page-blog.s3.amazonaws.com
 		- d13w15pduyzezm.cloudfront.net
 
-		- status-page-blog.herokuapp.com
-
-			- blog
-
-
-	Problematic subdomains:
-
-		- blog *
-		- meta
-
-	* Mismatched, CN: *.herokuapp.com
-
-
-	Fully covered subdomains:
-
-		- blog		(→ status-page-blog.herokuapp.com)
-		- hosted
-		- manage
-		- www
-		- *		(per-client subdomains)
-
 -->
 <ruleset name="StatusPage.io">
 
 	<target host="statuspage.io" />
 	<target host="*.statuspage.io" />
 	<target host="metastatuspage.com" />
-	<target host="status-page-blog.herokuapp.com" />
```

Comment:
Outdated probably isn't the proper term for this one. But it is already covered in [Heroku.xml](https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/Heroku.xml).